.env

```diff
@@ -0,0 +1,4 @@
+DB_HOST=127.0.0.1
```

Comment:
Fix the `.env` showing up credentials here. It is very critical that all project secrets or credentials are never committed at all. So ensure that as part of your repo setup, you setup common files to gitignore.